.github/workflows/sdk.yml

```diff
@@ -262,6 +262,8 @@ jobs:
         run: |
           python3 -m easy_install pip
           pip install setuptools wheel twine
+          pip install urllib3==1.26.6
```

Comment:
reason ?

Reply:
twine run failed with higher urllib version https://github.com/4paradigm/OpenMLDB/actions/runs/5407689152/jobs/9829475227

Reply:
maybe caused by #3284 . It is not because urllib too old, instead, twine is too new. 
newer urllib will not work on centos7 because libssl too old. We should lock the twine version. 
@vagetablechicken

Reply:
ghcr.io/4paradigm/hybridsql:latest doesn't has them, so we should install it or them(locked version) in the image?

Reply:
Looks like urllib3 >=2.0.0 breaks twine, refer https://github.com/pypa/twine/issues/989

Reply:
> pip install twine "urllib3>=1.26.0,<2.0.0"

Reply:
twine >= 3.8.0 require urllib3>=1.26.0, so it 2.x urllib.
3.7.0 twine works well